src/Compilers/CSharp/Portable/Symbols/NamedTypeSymbol.cs

```diff
@@ -617,6 +617,12 @@ public override TResult Accept<TResult>(CSharpSymbolVisitor<TResult> visitor)
             }
         }
 
+        // PROTOTYPE(NullableReferenceTypes): Combine with other GetDeclaredBaseType overload and make abstract.
+        internal virtual NamedTypeSymbol GetDeclaredBaseType(ConsList<Symbol> basesBeingResolved, bool ignoreNullability)
```

Comment:
>GetDeclaredBaseType [](start = 41, length = 19)

Where is this method called from? It seems that only the overload in PENamedTypeSymbol is called from PENamedTypeSymbol itself.

Reply:
Removed method from base type, thanks.

---
In reply to: [198267943](https://github.com/dotnet/roslyn/pull/28028#discussion_r198267943) [](ancestors = 198267943)